host_files/herbert/nginx/sonarqube.zeus.gent

```diff
@@ -0,0 +1,37 @@
+upstream cat {
+   server 10.1.0.210:9000;
+   keepalive 32;
+}
```

Comment:
Pretty sure this should also be changed to `sonarcube` then, and then maybe double check of the upstream IP is correct